File: JavaScript projects/JavaScript Basics/06.Nested Loops/Exercises/01_number_pyramid.js
function pyramid(number){
    let row = 1;
    let n = 1;
    let output = "";

    while (n != (number + 1)){
        for (let col = 1; col <= row; col++){
            output += `${n} `
            n++;
        if (n === (number +1))break;
        }
        row ++;
        output += `\n`
    }
    console.log(output)
}
pyramid(7);
pyramid(12);
pyramid(15);

File: Python projects/Python Basics/06_nested_loops/exercises/01_number_pyramid.py
number = int(input())
counter = 0

for row in range(1, number + 1):
    for column in range(1, row + 1):
        counter += 1
        print(f"{counter}", end=" ")
        if counter == number:
            break
    if counter == number:
        break
    print()
